x-pack/plugins/observability_shared/public/hooks/use_get_user_cases_permissions.tsx

```diff
@@ -19,11 +20,11 @@ export function useGetUserCasesPermissions() {
     delete: false,
     push: false,
   });
-  const uiCapabilities = useKibana().services.application.capabilities;
+  const uiCapabilities = useKibana().services.application!.capabilities;
 
-  const casesCapabilities = useKibana().services.cases.helpers.getUICapabilities(
-    uiCapabilities[casesFeatureId]
-  );
+  const casesCapabilities = useKibana<{
+    cases: CasesUiStart;
```

Comment:
Nit: add `CasesUiStart` to interface `ObservabilitySharedStart` and use that interface here